app/mailers/provider_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class ProviderMailer < ApplicationMailer
+  def request_created_email
+    @provider = params[:linked_token].entity.provider
+    @request_url = 'localhost:3000/' + 'request/' + params[:linked_token].entity.id.to_s
```

Comment:
Probably going to need a way to make this base url configurable?

Or can you get the current domain from the request somehow?

Reply:
Good point! I was lazy here as I'm not too familiar with how Rails environment variables work and I wasn't sure if the app would be hosted on feedadoc.com or some dev iteration yet. I'll convert url definition into some sort of service and maybe we can circle back to update with environment vars later? Unless someone wants to school me on what that looks like in Rails land.

Reply:
Just to wrap this conversation, I'm going to merge as is and circle back to this elsewhere. Great call out, though. Thanks!